mu/virtual_environment.py

```diff
@@ -370,15 +375,18 @@ def run_subprocess(self, *args, **kwargs):
         process = subprocess.run(
             list(args),
             stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
+            stderr=subprocess.PIPE,
             **kwargs
         )
+        output = process.stdout.decode(sys.stdout.encoding, errors="replace") + process.stderr.decode(sys.stderr.encoding, errors="replace")

```

Comment:
Maybe we can also add a short string in front of stderr that makes searching in logs easier?

Reply:
Done